src/android/FileTransferBackground.java

```diff
@@ -319,7 +328,7 @@ private void addUpload(JSONObject jsonPayload) {
     private void startUpload(final String uploadId, final Data payload) {
         Log.d(TAG, "startUpload: Starting work via work manager");
 
-        OneTimeWorkRequest workRequest = new OneTimeWorkRequest.Builder(UploadTask.class)
+        OneTimeWorkRequest.Builder workRequestBuilder = new OneTimeWorkRequest.Builder(UploadTask.class)
                 .setConstraints(new Constraints.Builder()
                         .setRequiredNetworkType(NetworkType.CONNECTED)
                         .build()
```

Comment:
Samsung Devices: Uploading 300 images at a time blocks queue of uploads
On Pixel Devices works Fine
Removing Constraints in the WorkRequest helps to allow upload the image normally